### ISSUE TITLE
Global picture & other official flows updates

### ISSUE THREAD

**Issue by epicallan** [devinit/datahub#145]
- [x]  Where a reference exists on the page to "other official flows", for example, the clickable link to the visualisation (http://data.devinit.org / http://212.111.41.68:8080/ → Other Visualisation → Unbundling **other official flows**), the visualisation heading (http://data.devinit.org/unbundling-other-flows / http://212.111.41.68:8080/unbundling-other-flows) etc., please change to "**other official flows (non export credit)**". This is because "the exclusion of export credits will need to be made clear", see https://github.com/devinit/ddw-data/issues/153#issuecomment-343143990 for details. 

~~@Duncan-Knox, please provide instructions for how to change the following line:~~
 ~~- Explore and compare funding priorities for other official flows (OOFs)~~

Not needed.

**Comment by Duncan-Knox** [comment]
Hi @epicallan I'll begin checks now - first points 

[ ] Sub-title states '2012 prices' please update to say '2015 prices'
[ ] once explore is clicked, the data on expenditure is not showing - it just states US$0. 

Not if the second point above is on purpose, however to progress with checks, I'll need to see the figures.

**Comment by niklasrie** [comment]
Hi @epicallan here is my feedback on the data I uploaded and a few general things I noted. It may look a lot, there are some minor issues though, which might not need fixing immediately. Hence I split my feedback and high and low priority.

## **High priority**

### General 
- [ ] Methodology and data section seems outdated, with old decriptions/sources/datasets on github. Please update according to new/updated datasets. 

### Fragile states
- [ ] Please update scale/range, which is a remnant of old dataset. For the new data we only require two shades of red (fragile, extremely fragile = 1, 2) and grey (stable / not applicable / no data). This should fix the error on landing map, which shows all fragile states as either "sustainable" or "very sustainable" (equal to 1 and 2 on old scale). 
- [ ] Please update description (old) to: "The OECD Fragiliy Framework identifies countries are fragile or extremely fragile based on 50 indicators across five dimensions (economic, environmental, political, security and societal). It is part of the OECD report 'States of Fragility'."

### Forgotten Crises 
- [ ] Please update start year to 2004 and end year to 2017 and please add data for the respective additional years. 
- [ ] Please add following sentence to the description: "The map shows countries' number of appearances on the index since then." 

### Humanitarian assistance
- [ ] Please update start year to 2000 and end year to 2015 and please add data for the respective additional years. 
- [ ] Was my mistake not to delete all 0 values in dataset. These show up as light blue shading and not grey, which visually suggests a few donor countries frequently received humanitarian assistance. Will update dataset and get it you, for you to update it on the hub.  

## **Low priority**

### General
- [ ] Please update country names on landing map for those without data, currently spelled lowercase and with hyphens instead of spaces

### Fragile states
- [ ] Suggest to drop top and bottom 10 for this section, as it does not make much sense given the limited range (and is misleading for bottom 10 even). 

### Human hazard and exposure score
- [ ] Suggest differentiating the shading on the map more. Please extend scale at bottom end to 1 and smaller, in steps of integers. We lose granularity of the underlying data and provide a slightly misleading visual otherwise. 

### Natural hazard and exposure score
- [ ] Please show data values on landing map to 1 decimal point (same as underlying data), as rounded values make it difficult to understand changes in shading year-to-year. 
- [ ] Suggest differentiating the shading on the map more. Please extend scale at bottom end to 1 and smaller, in steps of integers. (see above for reasoning)

### Number of people displaced due to human hazard by hosting country
- [ ] For countries with values less than 1000, please remove the decimal point, as values represent a headcount and only come in integers. 

If there are any questions or anything else left for me to check, please let me know. And thank you very much for all of the work on the DDH Allan!

**Comment by JordanDI** [comment]
Hello @epicallan 

I've a suggestion for the presentation of the climate vulnerability data, on page http://212.111.41.68:8080/ > Vulnerability >Climate vulnerability ...

- [ ] Add another decimal place to the values presented in the top and bottom tables, as well as the values which appear when hovering over a country

Thanks

**Comment by Duncan-Knox** [comment]
Hi @epicallan 

@Emily-JM has checked the unbundling OOFs data in detail and can confirm there are no issues with the data. The checks were conducted both on the staging site and the live website. However two minor points around usability came up:

- [ ] How easy is it to remove all the drop down options which receive 0 disbursements over the entire period? e.g. from: 'United Nations Children's Fund (UNICEF)'. or sector 'other sectors [grouped].
- [ ] The toggling system re-sets itself when the year is changed, is this a quick fix to prevent this from happening?


Neither of these are high priority.

**Comment by niklasrie** [comment]
Hi @epicallan and happy New Year :-) (is that still admissible to say?) 

I just had a quick check through the updated DDH, and realised the following: 

- [ ] Under Humanitarian finance -> Humanitarian assistance we show the time horizon 2010 to 2015. If I remember correctly, I prepared 2000 to 2015 for the update (thinking that given we have the data going back further, we might as well show it). This is also what I updated in the metadata. There might be a technical reason why we decided not to go back to 2000, but if not, would it be possible to update that? 

- [ ] It seems that all values across all sections are now being displayed in full and with 2 dps, without thousand separators etc. as a string of numbers. This is quite difficult to read, so would it be possible to revert back to the original way of displaying all of the figures? 

Thanks so much and best regards!
